demos/index.html

```diff
@@ -44,6 +44,7 @@
           <li><a href="snackbar.html">Snackbar</a></li>
           <li><a href="textfield.html">Text Field</a></li>
           <li><a href="theme.html">Theme</a></li>
+          <li><a href="toolbar/index.html">Toolbar</a></li>
```

Comment:
Since fixed top is implemented through `position:fixed`, we need to keep it in a separate page in order to demo it. Currently, I am using iframe to embed them to get a direct visual idea for the user. I will also add a link to let user view in full page mode and make it clear that demo is using iframe.